src/Exadel.CrazyPrice/Exadel.CrazyPrice.IdentityServer/Options/AccountOptions.cs

```diff
@@ -14,7 +14,6 @@ public class AccountOptions
 
         public static bool ShowLogoutPrompt = true;
         public static bool AutomaticRedirectAfterSignOut = true;
-
-        public static string InvalidCredentialsErrorMessage = "Invalid username or password";
+        public static string InvalidCredentialsErrorMessage = "Invalid_email_or_password";
```

Comment:
Rename to InvalidCredentialsErrorMessageKey to identify that message will be looked up